<!-- -*- mode: XML -*- -->
<!-- run velodyne_pointcloud/CloudNodelet in a nodelet manager -->

<launch>
  <arg name="calibration" default="" />
  <arg name="manager" default="velodyne_nodelet_manager" />
  <arg name="max_range" default="130.0" />
  <arg name="min_range" default="0.9" />
  <arg name="num_points_threshold" default="300"/>
  <arg name="invalid_intensity" default="[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]"/>

  <node pkg="nodelet" type="nodelet" name="$(arg manager)_cloud" args="load velodyne_pointcloud/CloudNodelet $(arg manager)">
    <param name="calibration" value="$(arg calibration)"/>
    <param name="max_range" value="$(arg max_range)"/>
    <param name="min_range" value="$(arg min_range)"/>
    <param name="num_points_threshold" value="$(arg num_points_threshold)"/>
    <param name="invalid_intensity" value="$(arg invalid_intensity)"/>
  </node>
</launch>
